code/obj/sealab_objects.dm

```diff
@@ -172,6 +172,115 @@
 	icon_state = "tubesponge_small"
 	database_id = "sea_plant_tubesponge-small"
 
+//NADIR DOODADS (indev)
```

Comment:
Ordinarily I would say 'uncomment or remove' for the commented code but if this is being worked on then it's fine